plugins/maciVoting/hooks/useCoordinatorFinalize.tsx

```diff
@@ -0,0 +1,34 @@
+import { useCallback, useState } from "react";
```

Comment:
Created `useCoordinatorFinalize` hook to have separate `finalizeStatus` per each poll